sphinx_gallery/gen_gallery.py

```diff
@@ -104,6 +104,7 @@ def __call__(self, gallery_conf, script_vars):
     'prefer_full_module': [],
     'api_usage_ignore': '.*__.*__',
     'show_api_usage': False,
+    'copyfile_regex': None,
```

Comment:
Perhaps instead:
```suggestion
    'copyfile_regex': '',
```
I think it's more consistent with our "string but don't use" of `ignore_repr_types` at least. There might be some advantage to having it always be a string in terms of sphinx detecting value types, too